src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -447,7 +447,7 @@ public function getProtocolVersion()
      * If the status text is null it will be automatically populated for the known
      * status codes and left empty otherwise.
      *
-     * @return Response
+     * @return $this
```

Comment:
I personally use `static` everytime, but according to [PSR-5 Keywords section, 14-15:](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#keyword)

> `static`, the element to which this type applies is of the same class as which the documented element is contained, or when encountered in a subclass is of type of that subclass instead of the original class.
>
>This keyword behaves the same way as the keyword for late static binding (not the static method, property, nor variable modifier) as defined by PHP.
>
>`$this`, the element to which this type applies is **the same exact instance as the current class in the given context**. As such this type is a **stricter version of static** as, in addition, **the returned instance must not only be of the same class but also the same instance**.

`$this` seems the most appropriate. 👍 

`self` isn't because:
> the element to which this type applies is of the same class as which the documented element is originally contained

which means:

> it may lead to confusing situations when inheritance is involved